homeassistant/components/lcn/const.py

```diff
@@ -149,7 +150,7 @@
     "LCN",
     "NATIVE",
     TEMP_CELSIUS,
-    "°K",
+    f"{UNIT_DEGREE}K",
```

Comment:
I don't think this one is valid. This is a temperature in kelvin. TEMP_KELVIN is more in place here?

Reply:
Good idea, added this constant and used it there.